composer.json

```diff
@@ -17,7 +17,7 @@
     },
     "require": {
         "neos/neos-development-collection": "2.0.x-dev",
-        "neos/flow-development-collection": "@dev",
+        "neos/flow-development-collection": "3.0.x-dev",
```

Comment:
But i’d vote still for get the correct versions in to be a) safe and b) make clear wich versions work together